test/discovery_failure_test.py

```diff
@@ -23,7 +23,9 @@ def test_discover_test_with_broken_import(self):
         non-existent module is discovered."""
 
         stdout, stderr = cmd_output(
-            'python', '-m', 'testify.test_program', 'discovery_error', cwd='examples',
+            'python', '-W', 'ignore::RuntimeWarning:runpy:',
```

Comment:
curious: what's the `RuntimeWarning` that's happening here -- can we fix the code instead of silencing the warning?